guides/cloud/aws_sandbox.md

```diff
@@ -31,7 +31,7 @@ Reasons for using this tool can be found [here](https://github.com/99designs/aws
 2. Add the following config to your `~/.aws/config` file.
 
 ```
- [profile madesso]
+ [profile mt-playground]
```

Comment:
Should this profile name change be reflected in the example aws-vault command on lines 42 & 44?